pyproject.toml

```diff
@@ -75,6 +75,9 @@ vertex = ["docker", "kfp", "google-cloud-aiplatform"]
 sagemaker = ["sagemaker", "boto3"]
 docker = ["docker"]
 
+all = ["dask", "dask-cuda", "s3fs", "adlfs", "gcsfs", "docker", "kfp", "google-cloud-aiplatform",
```

Comment:
Guess this is useful for local development in case you want to install all dependencies. Should we adjust the readme?

Reply:
For local development, this can still be done with `poetry install --all-extras`. I just needed this to install all extras when installing from the wheel, which I think is only needed in our CI/CD.